joern-cli/src/main/scala/io/shiftleft/joern/Predefined.scala

```diff
@@ -0,0 +1,33 @@
+package io.shiftleft.joern
+
+object Predefined {
+
+  /* ammonite tab completion is partly broken for scala > 2.12.8
+   * applying workaround for package wildcard imports from https://github.com/lihaoyi/Ammonite/issues/1009 */
+  val shared: String = """
+        |import gremlin.scala.{`package` => _, _}
+        |import io.shiftleft.console.{`package` => _, _}
```

Comment:
@fabsx00 Generally if one wants to import the whole package object we would simply write
```
import io.shiftleft.console._
```